ROADMAP.md

```diff
@@ -0,0 +1,171 @@
+# Babel Roadmap
+
+> Not everything is set in stone or has an issue for it! Looking to post these to get more people involved or integrate with other projects.
+
+## Ecosystem
+
+### Remake `compat-table` used in preset-env
+
+> https://github.com/kangax/compat-table could use a remake, ideally work with browser vendors on this
+    
+- There is also https://github.com/mdn/browser-compat-data
+- Also use data from test262?
+- Run tests against real browsers
+- have a data-only format
+- Need continued maintainence
+
+### Polyfill behavior
+
+> This is regarding https://github.com/babel/babel/tree/master/packages/babel-preset-env#usebuiltins-usage
+
+- Allow any substitute polyfill instead of `core-js`. You should be able to override anything (custom `Promise`, etc)
+- Make `"usage"` option the default after it is stable.
+
+### Build/publish workflow
+  - Guide on compiling/publishing ES2015+, .mjs, etc: https://twitter.com/philwalton/status/908082461799616512
+  - Support multi-build/folder outputs?
```

Comment:
That's interesting and probably useful for some, but I kinda feel that i.e. `rollup` has this area covered for most of use cases - especially that it understand the dep graph, so it can just nicely output flat bundles and all that jazz.

Reply:
I agree, it's not so common in JS but CMake is a super tool for emitting multiple binaries from a monorepo-ish project. 

One issue is that npm/node doesn't support super good a flat bundle. React-dom for example has to put a lot of files in the root of the project https://unpkg.com/react-dom@16.2.0/.

Reply:
In this regard flat bundles and multi-file directories doesnt really differ that much - if one needs multiple entry points for the package.

But actually many use cases are covered automatically - you just have to output multiple flat bundles and make use of `main`, `module`, `browser`, `react-native` fields in `package.json`.